apps/node-pl/particle.app.config.js

```diff
@@ -18,7 +18,7 @@ const APP_DESIGN_SYSTEM = path.resolve(__dirname, '../../source/default');
 // Where should this app compile to, e.g. dist/app-drupal/assets/
 const APP_DIST = path.join(PATH_DIST, APP_NAME, ASSETS_BUNDLE_FOLDER);
 // Base path for all assets
-const APP_DIST_PUBLIC = path.join(APP_NAME, ASSETS_BUNDLE_FOLDER);
+const APP_DIST_PUBLIC = `/${path.join(APP_NAME, ASSETS_BUNDLE_FOLDER)}`;
```

Comment:
😅

Reply:
It's **always** the single-character fixes ...